decoders/mil-std-1553/pd.py

```diff
@@ -0,0 +1,386 @@
+                        cmdinfvalue = self.m_accept[self.lang]
+                        if(adrvalue == 31):
+                            self.cwdetailscolour = 3
+                            cmdinfvalue = cmdinfvalue + self.e_m_notgroup[self.lang]
+                    elif (countdwvalue == 1):
+                        cmdinfvalue = self.m_synch[self.lang]
+                    elif (countdwvalue == 2):
+                        cmdinfvalue = self.m_sendaw[self.lang]
+                        if(adrvalue == 31):
+                            self.cwdetailscolour = 3
+                            cmdinfvalue = cmdinfvalue + self.e_m_notgroup[self.lang]
+                    elif (countdwvalue == 3):
+                        cmdinfvalue = self.m_selfcontrol[self.lang]
+                    elif (countdwvalue == 4):
+                        cmdinfvalue = self.m_blocktx[self.lang]
+                    elif (countdwvalue == 5):
+                        cmdinfvalue = self.m_unlocktx[self.lang]
+                    elif (countdwvalue == 6):
+                        cmdinfvalue = self.m_blocksign[self.lang]
+                    elif (countdwvalue == 7):
+                        cmdinfvalue = self.m_reset[self.lang]
+
+                    elif (countdwvalue == 16):
+                        cmdinfvalue = self.m_sendvec[self.lang]
+                        if(adrvalue == 31):
+                            self.cwdetailscolour = 3
+                            cmdinfvalue = cmdinfvalue + self.e_m_notgroup[self.lang]
+                    elif (countdwvalue == 18):
+                        cmdinfvalue = self.m_lastcomm[self.lang]
+                        if(adrvalue == 31):
+                            self.cwdetailscolour = 3
+                            cmdinfvalue = cmdinfvalue + self.e_m_notgroup[self.lang]
+                    elif (countdwvalue == 19):
+                        cmdinfvalue = self.m_sendvsk[self.lang]
+                        if(adrvalue == 31):
+                            self.cwdetailscolour = 3
+                            cmdinfvalue = cmdinfvalue + self.e_m_notgroup[self.lang]
+
+                else:
+                    if (countdwvalue == 17):
+                        cmdinfvalue = self.m_synchdw[self.lang]
+                    elif (countdwvalue == 20):
+                        cmdinfvalue = self.m_blocki[self.lang]
+                    elif (countdwvalue == 21):
+                        cmdinfvalue = self.m_unlocki[self.lang]
+                cwdetailsvalue = cwdetailsvalue + cmdinfvalue
+
+            else:
+                if kvalue == 1:
+                    cwdetailsvalue = cwdetailsvalue + self.m_transmit[self.lang]
+                else:
+                    cwdetailsvalue = cwdetailsvalue + self.m_reception[self.lang]
+
+                if countdwvalue == 0:
+                    countdwvalue = 32
+                cwdetailsvalue = cwdetailsvalue + ' ' + format(countdwvalue) + self.m_dw[self.lang]
+            pass    
```

Comment:
Is this pass intended or a bug? There's a "return cwdetailsvalue" that follows immediately after, so am I understanding correctly that the pass was used for debugging and should be removed?